crates/router/src/connector/stripe/transformers.rs

```diff
@@ -2125,23 +2171,42 @@ impl TryFrom<&types::PaymentsCaptureRouterData> for CaptureRequest {
     }
 }
 
-impl TryFrom<&types::PaymentsPreProcessingRouterData> for StripeAchSourceRequest {
+impl TryFrom<&types::PaymentsPreProcessingRouterData> for StripeCreditTransferSourceRequest {
     type Error = error_stack::Report<errors::ConnectorError>;
     fn try_from(item: &types::PaymentsPreProcessingRouterData) -> Result<Self, Self::Error> {
-        Ok(Self {
-            transfer_type: StripePaymentMethodType::AchCreditTransfer,
-            payment_method_data: AchBankTransferData {
-                email: connector::utils::PaymentsPreProcessingData::get_email(&item.request)?,
-            },
-            currency: item
-                .request
-                .currency
-                .get_required_value("currency")
-                .change_context(errors::ConnectorError::MissingRequiredField {
-                    field_name: "currency",
-                })?
-                .to_string(),
-        })
+        let currency = item.request.get_currency()?.to_string();
+
+        match &item.request.payment_method_data {
```

Comment:
`item` is already a reference, you don't need the `&` again.

Reply:
Removing `&` throws an error